src/cc65/main.c

```diff
@@ -951,6 +962,9 @@ int main (int argc, char* argv[])
 
                 case 'O':
                     IS_Set (&Optimize, 1);
+                    IS_Set (&CodeSizeFactor, 100);
+                    IS_Set (&EnableRegVars, 0);
+                    IS_Set (&InlineStdFuncs, 0);
```

Comment:
I agree with mrdudz: -O, -Oi, -Os and -Or are different option which just happen be be related to optimization and and are therefore allowed to be written as -Oisr. But I don't see their setting to influence each other.